src/style.css

```
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+h1,
+h2,
+h3,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --btn-color: #00acdc;
+
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  line-height: 18px;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  color: var(--secondary-color);
+  font-size: 10px;
+  line-height: 13px;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.card__review-stars {
+  display: flex;
+}
+
+.card__review-star {
+  width: 16px;
+  height: 16px;
+  background: url("/images/star.svg") no-repeat center;
+}
+
+.card__review-star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__review-stars .card__review-star:nth-child(-n + 4) {
+  background: url("/images/star-active.svg") no-repeat center;
+}
+
+.card__price {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__review-review {
+  color: var(--main-color);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-titel {
+  font-size: 12px;
+  line-height: 18px;
+}
```

```suggestion
color: var(--secondary-color);
}
```

Отут ще не додав колір.